src/hotspot/share/jvmci/jvmciJavaClasses.hpp

```diff
@@ -57,7 +57,7 @@
                          jvmci_method, \
                          jvmci_constructor) \
   start_class(Services, jdk_vm_ci_services_Services)                                                          \
-    jvmci_method(CallStaticVoidMethod, GetStaticMethodID, call_static, void, Services, initializeSavedProperties, byte_array_void_signature, (JVMCIObject serializedProperties)) \
```

Comment:
The final arg for the `jvmci_method` macro is never used so I removed it everywhere.